_data/projects/apps-android-commons.yml

```diff
@@ -0,0 +1,11 @@
+name: Wikimedia Commons Android App
+desc: The Wikimedia Commons Android app allows users to upload pictures from their Android phone/tablet to Wikimedia Commons. Initially started by the Wikimedia Foundation, this app is now maintained by volunteers. Anyone is welcome to improve it, just choose among the open issues and send us a pull request :-)
```

Comment:
Can we trim this sentence to drop the last bit, so that it doesn't take up too much real estate on the site? 

> Anyone is welcome to improve it, just choose among the open issues and send us a pull request :-)

The listing on the site is evidence enough that you want others to get involved.